tests/rkt_tests.go

```diff
@@ -171,16 +175,34 @@ func expectTimeoutWithOutput(p *gexpect.ExpectSubprocess, searchString string, t
 	return expectCommon(p, searchString, timeout)
 }
 
-func patchTestACI(newFileName string, args ...string) {
+func patchTestACI(newFileName string, args ...string) string {
```

Comment:
@krnowak @iaguis : this function now returns the **new** file path, so callers are expected to use the returned value. It was changed in other tests but not in the isolators tests. I will fix that now and close #1104 :)